daemon/cmd/daemon_main.go

```diff
@@ -297,10 +297,14 @@ func InitGlobalFlags(cmd *cobra.Command, vp *viper.Viper) {
 
 	flags.Var(option.NewNamedMapOptions(option.IPAMMultiPoolPreAllocation, &option.Config.IPAMMultiPoolPreAllocation, nil),
 		option.IPAMMultiPoolPreAllocation,
-		fmt.Sprintf("Defines the minimum number of IPs a node should pre-allocate from each pool (default %s)", defaults.IPAMMultiPoolPreAllocation))
-	vp.SetDefault(option.IPAMMultiPoolPreAllocation, defaults.IPAMMultiPoolPreAllocation)
+		fmt.Sprintf("Defines the minimum number of IPs a node should pre-allocate from each pool (default %s)", defaults.IPAMDefaultIPPool))
```

Comment:
This needs to be:

```suggestion
		fmt.Sprintf("Defines the minimum number of IPs a node should pre-allocate from each pool (default %s)", ""))
```

Also, please don't forget to run `make -C Documentation update-cmdref` afterwards

Reply:
Don't we need to display the default value? Currently, `apam-multi-pool-pre-allocation` still has a default value.

Reply:
Oh, right I misread the code. However, we should display the =8 value still, i.e.
```suggestion
		fmt.Sprintf("Defines the minimum number of IPs a node should pre-allocate from each pool (default %s=8)", defaults.IPAMDefaultIPPool))
```